parachain/runtime/testnet-interlay/src/lib.rs

```diff
@@ -1481,15 +1481,15 @@ impl_runtime_apis! {
         Balance,
     > for Runtime {
         fn get_account_liquidity(account: AccountId) -> Result<(Liquidity, Shortfall), DispatchError> {
-            Loans::get_account_liquidity(&account)
+            Loans::get_account_liquidity(&account).map(|liquidity| liquidity.to_rpc_tuple())
```

Comment:
This mapping is required because the `Amount<T>` type within `AccountLiquidity<T>` isn't available as a type in the primitives crate. If there's another way to use it here let me know.